tools/webpack-core/src/plugins/IModeljsLibraryImportsPlugin.ts

```diff
@@ -6,12 +6,32 @@
 import { Compiler } from "webpack";
 import { ConcatSource } from "webpack-sources";
 import * as utils from "./IModeljsLibraryUtils";
-
 function getLoaderString(pkgName: string, pkgVersion: string) {
-  return `  module.exports = (() => {
+  return `  const semverGte = (appVersion, extensionVersion) => {
+    const dashIndexApp = appVersion.indexOf("-");
+    const dashIndexExtension = extensionVersion.indexOf("-");
+    if (dashIndexExtension >= 0 ) {
+      console.warn("This extension has a dependency on a dev version. Use in prod with caution.");

```

Comment:
Can we switch this to be a bit more explicit about the possible complications?

`The Extension '${Extension Name}' depends on a nightly version of iModel.js Shared Libraries.  Use in production with caution as stability is not guaranteed.`

Reply:
Do you know how I'd get access to the extension name?

Reply:
Not sure the best way to get it so lets skip it in this PR.